src/main/java/com/checkout/Environment.java

```diff
@@ -6,18 +6,27 @@
 
 public enum Environment {
 
-    SANDBOX(create("https://api.sandbox.checkout.com/"), create("https://files.sandbox.checkout.com/"), create("https://access.sandbox.checkout.com/connect/token")),
-    PRODUCTION(create("https://api.checkout.com/"), create("https://files.checkout.com/"), create("https://access.checkout.com/connect/token"));
+    SANDBOX(create("https://api.sandbox.checkout.com/"),
+            create("https://files.sandbox.checkout.com/"),
+            create("https://transfers.sandbox.checkout.com/"),
+            create("https://access.sandbox.checkout.com/connect/token")),
+    PRODUCTION(create("https://api.checkout.com/"),
+            create("https://files.checkout.com/"),
+            create("https://files.checkout.com/"),
```

Comment:
It must be `https://transfers.checkout.com/`.

Reply:
Nice catch. thank you!